examples/lib/versionManagement.js

```diff
@@ -0,0 +1,86 @@
+// Get the current version number from package.json on GitHub.
+function getLatestVersionNumber(callback) {
+  var request = new XMLHttpRequest();
+  request.onreadystatechange = function() {
+    if(request.readyState == 4 && request.status == 200) {
+      var response = JSON.parse(this.responseText);
+      var latestVersionNumber = JSON.parse(atob(response.content)).version;
+      console.log("The latest NPM version number from GitHub is: " + latestVersionNumber);
+      callback(latestVersionNumber);
+    }
+  }
+  request.open("GET", "https://api.github.com/repos/publiclab/image-sequencer/contents/package.json", true);
```

Comment:
This will GET the latest package JSON. It may not be the one online. We need a better way of GET-ing the version number.

Reply:
By online, do you mean at sequencer.publiclab.org? If so, it looks like we could get the version number from the `stable` branch here on GitHub. 

Whether we do this depends on whether offline users are mainly working off the `stable` branch or the `main` branch. The current implementation of this function is getting the version number from the `main` branch, which may be what people are using offline anyways.

Reply:
For beta.sequencer.publiclab.org, we'll have to get the version from that repo. Beta is on a separate repo btw. For the stable one, it will bw different. Anyway, it will be a mess. Can't we get it from `current_location/package.json` or something? Make it dynamic instead of hard coded?